_common/MarkdownHost.php

```diff
@@ -95,12 +95,14 @@ function __construct($file) {
 
       // Performs the parsing + prettification of Markdown for display through PHP.
       $Parsedown = new \ParsedownExtra();
```

Comment:
Is $Parsedown still needed?

Reply:
I think no, @darcywong00. I will remove it since nothing is breaking while testing.